composer.json

```diff
@@ -26,7 +26,8 @@
     "require": {
         "php": "~5.4",
         "ext-openssl": "*",
-        "namshi/jose": "~4.0"
+        "namshi/jose": "~4.0",
+        "willdurand/negotiation": "1.3.4"
```

Comment:
The library README recommended picking a version, but looking at this I can see we've used ~5.4 for PHP and ~4.0 for namshi/jose. 

@brianjmiller do you have an opinion?